Spatial.md

```diff
@@ -371,8 +371,8 @@ Handling spatial data
 
 ### Spatial sampling
 
--   `r pkg("spsurvey")` provides a range of sampling
-    functions.
+-   `r pkg("spsurvey")` provides functions to select GRTS probability samples
```

Comment:
@michaeldumelle could you explain the GRTS abbreviation here?

Reply:
@Nowosad explained the GRTS abbreviation in the most recent commit thanks!